test/Driver/loaded_module_trace_multifile.swift

```diff
@@ -10,10 +10,10 @@
 // CHECK: "name":"loaded_module_trace_multifile"
 // CHECK: "arch":"{{[^"]*}}"
 // CHECK: "swiftmodules":[
-// CHECK: "{{[^"]*}}/Module2.swiftmodule"
-// CHECK: "{{[^"]*}}/Module.swiftmodule"
-// CHECK: "{{[^"]*}}/Swift.swiftmodule"
-// CHECK: "{{[^"]*}}/SwiftOnoneSupport.swiftmodule"
+// CHECK: "{{[^"]*[/\\]}}Module2.swiftmodule"
+// CHECK: "{{[^"]*[/\\]}}Module.swiftmodule"
+// CHECK: "{{[^"]*[/\\]}}Swift.swiftmodule"
+// CHECK: "{{[^"]*[/\\]}}SwiftOnoneSupport.swiftmodule"
```

Comment:
This looks incorrect; this file is supposed to be JSON, right?

Reply:
Should be correct, it's looking for either a forwards or backslash, on windows the backslash gets escaped, so it'll be something like `filepath\\Module2.swiftmodule` which should match